llvm/lib/Transforms/Scalar/EarlyCSE.cpp

```diff
@@ -31,6 +31,7 @@
 #include "llvm/IR/Constants.h"
 #include "llvm/IR/Dominators.h"
 #include "llvm/IR/Function.h"
+#include "llvm/IR/IRBuilder.h"
```

Comment:
For some reason I thought we werent allowed to use builder in EarlyCSE, although thinking about the reason I don't have anythiny concrete.